build-ci/platforms.rst

```diff
@@ -0,0 +1,46 @@
+###################
+Supported Platforms
+###################
+
+Introduction
+============
+
+The LSST Data Management software is required to be portable across many compute platforms ranging from high-performance computing centres to laptops (DMS-REQ-0308).
+The software will likely work on any Unix-like operating system but we limit our testing and support to a fixed set of operating systems.
+
+Patches may be accepted from developers who test on other operating systems.
+
+Definitions
+===========
+
+We use the following definitions when defining which operating systems are supported.
+
+* A *supported* operating system is one which is regularly tested by our continuous integration system and which we will ensure all tests pass.
+  The list of supported operating systems may change over time, an in particular, if we encounter difficulties where keeping an operating system running is taking excessive developer and CI resources, we reserve the right to drop it from the list.
+  Each supported operating system will be associated with a specific compiler that we use for testing.
+  Other compilers may work but are not tested.
+* The *baseline* operating system is the operating system we are expecting to use for official deployments.
+  It is the primary operating system at NCSA and the software *must* work on it.
+  This operating system will be used for integration and verification testing.
+
+Supported Operating Systems
+===========================
+
+The following operating systems are supported and tested by Jenkins.
+
+CentOS 7
+--------
+
+The baseline operating system is CentOS7 and all code *must* work on CentOS 7.
+We currently use the default CentOS 7 compiler.
+We aim to support the most recent versions of CentOS7 within 6 months of release, although we can not guarantee that all NCSA computers and Jenkins will be upgraded promptly.
```

Comment:
Is it really useful to state this, given the caveat?

Reply:
At one point @billglick mentioned that it would be good to state our actual upgrade policy but I'm happy to drop it from here and find somewhere else to say it.